### DISCUSSION TITLE
[alpha4] - Show file path during upload/download in monitor mode

### DISCUSSION THREAD

**Discussion by bpozdena** [abraunegg/onedrive#2567] [Ideas]
Hi @abraunegg ,

I did a quick test of the alpha4 version (against your advise), just to test the integration with the GUI. 

The multi-threading progress output looks and works great for my needs. I'm just missing the relative path of the files while they are being downloaded/uploaded. This used to be printed in the previous versions and I was using it in the GUI to obtain details about the transferring file (size, icon, etc.). 

Would it be possible to print the relative path of the file together with the progress?
Example:
```
Downloading: /dir1/dir2/dir3/100MB.zip .................................. 80%   |  ETA   00:00:00
```

Alternatively, at least print the relative file path before the actual transfer starts, like in previous versions? This way, I could at least cache the path for later use. 
Example:
```
Downloading new file /dir1/dir2/dir3/100MB.zip ...
[...]
Downloading: 100MB.zip ...................................................... 80%   |  ETA   00:00:00
```

EDIT:
It would be great if the file path could also be printed when upload/download is being resumed (after interruption).

**Comment by abraunegg** [comment]
@bpozdena 
> Would it be possible to print the relative path of the file together with the progress?

I originally did this .. and pulled it to get feedback. Will revert that.

> Alternatively, at least print the relative file path before the actual transfer starts, like in previous versions? This way, I could at least cache the path for later use.

It currently prints the full path when it is complete.

> It would be great if the file path could also be printed when upload/download is being resumed (after interruption).

Odd .. it should be doing that ...... will investigate.

**Comment by abraunegg** [comment]
@bpozdena 
Also - please use `v2.5.0-alpha-4 GitHub version: v2.4.25-67-g2d9bdd1` ... some major issues were detected with Shared Folders use.
